packages/coin98-wallet/package.json

```diff
@@ -1,7 +1,27 @@
 {
   "name": "@near-wallet-selector/coin98-wallet",
   "version": "7.4.0",
+  "description": "Coin 98 wallet package for NEAR Wallet Selector.",
+  "keywords": [
+    "near",
+    "blockchain",
+    "wallets",
+    "dapps",
+    "near-protocol",
+    "near-blockchain",
+    "wallet selector",
+    "injected wallet",
+    "coin 98"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/near/wallet-selector/tree/main/packages/coin98-wallet"
```

Comment:
I think this URL can be set for all packages only to `https://github.com/near/wallet-selector.git` because all packages are in the same repo.